test.all.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
```

Comment:
```suggestion
#!/usr/bin/env sh
```
(here and in other shell scripts)

Reply:
Shebang is usually `#!/bin/sh`, but `#!/usr/bin/env something-other-than-sh`. `env` script does not improve portability in the case on `sh`. Strictly speaking, POSIX does not require neither /bin/sh, nor /usr/bin/env to present, so I'll stick with the way which looks for me as 'usually people doing it in this way'.